src/_locales/de/messages.json

```
@@ -0,0 +1,14 @@
+{
+  "extensionName": {
+    "message": "HoloSchedule",
+    "description": "Name of this extension which will be list in the store like AMO and Chrome Web Store"
+  },
+  "extensionDescription": {
+    "message": "Eine browser extension für alle Livestreams",
+    "description": "Description of this extension"
+  },
+  "browserActionTitle": {
+    "message": "Sei immer auf dem neusten Stand von HoloLive",
+    "description": "Title to show when user hover on the browser action button"
+  }
+}
```

Perserve all the empty lines at the end of the JSON files. (This will not result into an any error and is just my personal preference.)

```suggestion
}

```

---

Ok, i can add them easily back in :D